pom.xml

```diff
@@ -110,10 +110,16 @@
             <version>3.2.0</version>
         </dependency>
         <!-- https://mvnrepository.com/artifact/org.hibernate.orm/hibernate-core -->
-        <dependency>
+        <!--dependency>
             <groupId>org.hibernate.orm</groupId>
             <artifactId>hibernate-core</artifactId>
             <version>6.6.1.Final</version>
```

Comment:
different hibernate versions orm and without orm?

Reply:
6 commented out